_config.yml

```
@@ -73,6 +75,8 @@ contact_note: >
   You can even add a little note about which of these is the best way to reach you.
 
 google_analytics: UA-XXXXXXXXX # out your google-analytics code
+google_site_verification: # out your google-site-verification code (Google Search Console)
+bing_site_verification: # out your bing-site-verification code (Bing Webmaster)
 panelbear_analytics: XXXXXXXXX # panelbear analytics site ID
```

let's improve defaults of this entire block:
```suggestion
google_analytics:  # your google-analytics ID (format: UA-XXXXXXXXX)
google_site_verification:  # your google-site-verification ID (Google Search Console)
bing_site_verification:  # out your bing-site-verification ID (Bing Webmaster)
panelbear_analytics:  # panelbear analytics site ID (format: XXXXXXXXX)
```